src/pages/index.astro

```diff
@@ -43,8 +44,18 @@ await initializeWebContext(Astro, {
     <ElementColors />
     <Metadata entry={getEntry('metadata', 'main')} />
   </Fragment>
-  <Basics entry={getEntry('basics', 'main')} />
-  <Section class="gap-8">
+  <Sidebar>
+    <SidebarItem sectionId="basics" icon="fa6-solid:user">About me</SidebarItem>
+    <SidebarItem sectionId="skills" icon="fa6-solid:bars-progress">Skills</SidebarItem>
+    <SidebarItem sectionId="jobs" icon="fa6-solid:suitcase">Work experience</SidebarItem>
+    <SidebarItem sectionId="projects" icon="fa6-solid:rocket">Projects</SidebarItem>
+    <SidebarItem sectionId="references" icon="fa6-solid:comment">References</SidebarItem>
+    <SidebarItem sectionId="achievements" icon="fa6-solid:trophy">Achievements</SidebarItem>
+    <SidebarItem sectionId="education" icon="fa6-solid:graduation-cap">Education</SidebarItem>
+    <SidebarItem sectionId="favorites" icon="fa6-solid:star">Favorites</SidebarItem>
```

Comment:
Może tak miało być, ale brakuje Interests na side barze.

Reply:
It's intentional not to put interests into the sidebar. It showcases you can skip some sections in the sidebar if you consider them less important.